source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -282,7 +281,6 @@ const selectedFeatureLoaderStates = {
         CLOSE_POPUP: {
           actions: [
             assign({
-              mapLocationSelected: () => null,
```

Comment:
closing the popup shouldn't remove the marker - users need to be able to hide the popup and still see the marker.